src/content/docs/alerts/get-notified/notification-integrations.mdx

```diff
@@ -783,10 +785,16 @@ You can update ServiceNow attributes directly in the target table, such as **Inc
   
 ### Configure the Slack message template in workflows [#configure-slack-message-settings]
 
-  1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Workflows**</DNT> and click on an existing workflow, or the <DNT>**+ Add a workflow**</DNT> button.
-  2. Select a destination (Workspace) and a Slack channel to send messages. You can create a new destination if there is no pre-defined destination for the required workspace. For privacy reasons, users need to be authenticated once to select private channels. When selecting a private channel, the bot will automatically be added to the channel.
-  3. You can use the default notification or enrich it with custom details. Pick variables from the [variables menu](/docs/alerts-applied-intelligence/notifications/message-templates/#variables-menu) and apply [handlebars syntax](/docs/alerts-applied-intelligence/notifications/message-templates/#handlebars-syntax) to enrich your payload.
-  4. Click the <DNT>**Send test notification**</DNT> button to send a test notification with a pre-defined example payload to the channel. This creates a message in the selected Slack channel.
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Workflows**</DNT> and click on an existing workflow, or the <DNT>**+ Add a workflow**</DNT> button.
+2. Select a destination, also known as a workspace, and a Slack channel to send messages. You can create a new destination if there is no pre-defined destination for the required workspace. For privacy reasons, users need to be authenticated once to select private channels. When selecting a private channel, the bot will automatically be added to the channel.
```

Comment:
This might be awkward, but I almost feel like we should move "For privacy reasons..." to the top AHEAD of the procedural steps. Whether as raw text or a tip call out. It seems like something they should know ahead of getting into the procedure

Reply:
Good idea! I fixed this :)